meson.build

```diff
@@ -33,27 +33,39 @@ else
 	add_project_arguments('-DHAVE_ELOGIND=1', language : 'c')
 endif
 
+gdk = dependency('gdk-3.0', required: get_option('icon'))
+gdk_pixbuf = dependency('gdk-pixbuf-2.0', required: get_option('icon'))
+icon_feature = not get_option('icon').disabled()
+if icon_feature
+	add_global_arguments('-DSHOW_ICONS=1', language : 'c')
```

Comment:
I'd prefer to rename this to `HAVE_GDK_PIXBUF`

Reply:
Is it clear that this relates to having both gdk and gdk_pixbuf? They are largely independent things.

Reply:
Okay, maybe `HAVE_ICONS` then?

Reply:
Sounds good to me.